packages/teams-js/src/public/interfaces.ts

```diff
@@ -1262,3 +1262,41 @@ export interface ClipboardParams {
   /** Blob content in Base64 string format */
   content: string;
 }
+
+/**
+ * @hidden
+ * List of enter focus action items
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export enum EnterFocusActionItem {
+  /**
+   * Determines the direction to focus in app.
+   */
+  PreviousLandmark,
+  NextLandmark,
+  /**
```

Comment:
Maybe the comment could clarify how 'Read' and 'Compose' enumerations are different from the meaning and intent of PreviousLandmark and NextLandmark. And also the semantic meaning of 'Read' or 'Compose' w.r.t content of the app.

Reply:
makes sense, will add more details